projects/igniteui-angular/src/lib/combo/combo.component.ts

```diff
@@ -578,7 +579,16 @@ export class IgxComboComponent extends DisplayDensityBase implements IgxComboBas
      * ```
     */
     @Input()
-    public itemsMaxHeight = 480;
+    public get itemsMaxHeight(): number {
+        if (this._itemsMaxHeight === null || this._itemsMaxHeight === undefined) {
+            return this.itemHeight * 10;
```

Comment:
I would change this to:
this._itemsMaxHeight = this.itemHeight * 10;
this way we have single return.
10 could be const too.

Reply:
I agree that 10 should be a const - thanks for pointing it out!
We cannot do `this._itemsMaxHeight = this.itemHeight * 10`, as it would break the functionality - the next time we change item height, `this._itemsMaxHeight !== null` and we will not enter the `if` statement - the value won't change and the list will not scale to contain 10 items.